RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py

```diff
@@ -48,3 +48,7 @@
   MissCalibrate = False,
   ElectronPerADCGain = cms.double(600.) # it can be changed to something else (e.g. 135e) if needed
 )
+
+# to ensure reproducibility
```

Comment:
actually siPixelClusters DOES NOT run in cpu mode.
so this modification is actually required in the cpu workflow to be consistent with HLT & GPU code

Reply:
What is the reason for not using the full map on the GPU ?
"only" the memory usage, or also the (complexity of) the algorithm itself ?

I ask because since quite some time, HLT is thinking of moving to the full map - not because of any particular physics improvements, but to avoid one more calibration workflow.

Reply:
because is much more work. and as demonstrated by the MTV it makes little effect (to be verified on data)
open a issue. I do not think it is a show stopper for the demonstrator.
(shallow and wide prototype: using the full map is a typical going deep while we still need many other components)

Reply:
Yep, I agree we don't need it for the demonstrator.

Reply:
I've opened #27 as a reminder for the future.

Reply:
@VinInn, thanks, good point. Should we switch it for CPU by default in the patatrack branch then? (alternative being yet-another workflow and modifier)

Reply:
I do not want to add complexity, clearly even more in general, being able to compare HLT tracking (with HLT payloads) and offline HLT-like tracking (with the same payloads) is useful.
But  Ido not think that making all possible combination is wise: we may need just one "switch"
to use HLT-payloads offline and other to use off-line payloads at HLT...
I leave to the other to present different arguments...

Reply:
Since we want to do as much as possible a apple to apple comparison, I think we should use the `HLT` payloads for both CPU and GPU in the pixel-only tracking workflows: 10824.5, 10824.7, 10824.8, etc.

I can't say whether the change to 10824.5 should be only in the cms-patatrack branch or also upstream.

Reply:
Let's start then with `HLT` payload in cms-patatrack branch (and decide later if that is needed for upstream), I can make a PR.

Reply:
Thanks for #30 .